index.html

```diff
@@ -2434,6 +2434,58 @@ <h3>
           </p>
         </aside>
       </section>
+      <section class="informative">
+        <h3>
+          Web Apps with scope conflicts
+        </h3>
+        <p>
+          Because scopes are based on URL matching, it is possible for a developer to create
+          multiple web applications with the same, overlapping, or nested scopes. Doing so creates
+          several issues (detailed below) and is thus considered bad practice.
+        </p>
+        <ul>
+          <li>The scopes of the two web apps can be on the same origin. Not
+          recommended.
+          </li>
+          <li>The scope of one web app can be nested inside the scope of the
+          other. Strongly not recommended.
+          </li>
+          <li>The scopes of the two web apps can be the same. Strongly not
+          recommended.
+          </li>
+        </ul>
+        <p>
+          Same-origin scopes are not recommended due to origin-based settings
```

Comment:
This one doesn't quite make sense as written. It reads like we are discouraging same-origin scopes? It would be better to frame this explaining why one origin per application is ideal.

Reply:
Done. I could also call out examples of each of the list items being bad. e.g. Clearing storage for app A will clear storage for app B, Giving app A location permission possibly giving app B location permission, etc.